drivers/modem/modem_iface_uart.h

```diff
@@ -60,6 +60,10 @@ int modem_iface_uart_init(struct modem_iface *iface,
 			  struct modem_iface_uart_data *data,
 			  const char *dev_name);
 
+void modem_iface_sleep(struct modem_iface *iface);
+
+void modem_iface_wake(struct modem_iface *iface);
```

Comment:
Those functions are UART specific now, so I think they should be named `modem_iface_uart_sleep` and `modem_iface_uart_wake`.